pkgs/misc/tmux-plugins/default.nix

```diff
@@ -145,6 +145,24 @@ in rec {
     };
   };
 
+  extrakto = mkDerivation {
+    pluginName = "extrakto";
+    version = "unstable-2021-02-17";
+    src = fetchFromGitHub {
+      owner = "laktak";
+      repo = "extrakto";
+      rev = "45201c7331f9c2964f278df11cf5aba8dc466155";
+      sha256 = "0as8f4l9cqlq0haw95sagv7n83r3m817prjpn4yc5bh8x75adl25";
+    };
+    dependencies = [ pkgs.fzf pkgs.python3 pkgs.xclip ];
```

Comment:
This `dependencies` attribute is a fraud (I have been meaning to prune it from the file). Instead you have to take a similar approach outlined in [this comment](https://github.com/NixOS/nixpkgs/pull/95275/files#r522817362).

Reply:
Ok, I tried to follow the guides of that comment and I think I got it working now, could you do a quick review on that?

following the steps and ending with `tmux -f tmux.conf` worked, so I guess it's ok now, but I might be missing something and it may be picking the global one (it shouldn't, due to the nix-shell --pure, right?)